### ISSUE TITLE
output value question

### ISSUE THREAD

**Issue by oooo111** [evo-design/evo#97]
Hello, author. Could you please explain why the logits output by Evo are all negative and concentrated around the values of -20?
